region/maridia/inner-yellow/Bug Sand Hole.json

```diff
@@ -285,18 +250,25 @@
     {
       "id": 12,
       "link": [1, 3],
-      "name": "Space Jump Through the Door",
+      "name": "Space Jump Jumping Entry",
       "entranceCondition": {
-        "comeInJumping": {
-          "speedBooster": "any",
-          "minTiles": 0.4375
-        }
+        "comeInNormally": {}
       },
       "requires": [
+        "canPrepareForNextRoom",
         "SpaceJump",
-        "canPrepareForNextRoom"
+        {"or": [
+          {"doorUnlockedAtNode": 1},
+          {"doorUnlockedAtNode": 3}
+        ]}
```

Comment:
Come in space jumping, then the only way to land outside of the water is if either door can be opened with a shot or a Super. Regardless of which door you can land in, you can get to 3 (as you can then open any door, such as a PB door from 1).

It will still require unlocking 3 to go through it, so landing at 1 and unlocking 3 will still be accurate this way.